Solutions/Marain.Operations.StatusClient/Marain/Operations/Client/OperationsStatus/Models/Operation.cs

```diff
@@ -135,6 +135,10 @@ public virtual void Validate()
             {
                 throw new ValidationException(ValidationRules.CannotBeNull, "Status");
             }
+            if (TenantId == null)
```

Comment:
You can use `is null` pattern matching to avoid potential issues with operator== overloading etc.

Reply:
This is Autorest-generated code, so safer to leave as is I think.